docs/scripts/utils.py

```diff
@@ -32,9 +32,15 @@ def adjust_pictures(
 
     if line_index is not None:
         line = lines[line_index]
-        light_line = line.replace("reference external", "reference external only-light")
+        light_line = line
+        if "reference external" in line:  # for replacing the banner
+            light_line = line.replace(
+                "reference external", "reference external only-light"
+            )
+        if "_images/full_lookup_light.svg" in line:  # for the example in the README
```

Comment:
why is this special exception only done for the pic from the readme? does this `img ` pattern not exist ion other pictures? again the `replace` should not do anything if the pattern is not int he line, so an if clause (in this case even with a hardcoded searchpattern) shouldnt be necessary?

Reply:
Reason is that in the other md files that we have, we can properly include light/dark pictures in the .md file as described in https://pradyunsg.me/furo/reference/images/#different-images-for-dark-light-mode. But this is not possible for the README since this does not render properly on the github landing page since we show the .md file there and not the .html version that is created by sphinx.